config/settings.py

```diff
@@ -155,3 +160,16 @@
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
 AUTH_USER_MODEL = 'users.User'
+
+CORS_ALLOWED_ORIGINS = [
+    '<http://localhost:8000>',  # Замените на адрес вашего фронтенд-сервера
```

Comment:
Адрес указывается без `<>`
```suggestion
    'http://localhost:8000',  # Замените на адрес вашего фронтенд-сервера
```